extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzBuildTimeConfig.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.quartz.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public class QuartzBuildTimeConfig {
+    /**
+     * Enable cluster mode or not.
+     * <p>
+     * If enabled make sure to set the appropriate cluster properties.
+     */
+    @ConfigItem
+    public Optional<Boolean> clustered;
```

Comment:
This should probably just be a `boolean`.

Reply:
Yes, I wanted it to appear as non mandatory in the generated doc hence the usage of Optional.

Reply:
I think this is incorrect.  If it's optional then it has three possible states: "true", "false", and "empty".  We then need to document the behavior of all three states.

If it's a `boolean` then it is required, yes, but the default value will be `false`, so the user does not have to explicitly specify the value regardless.  If we want to regard such properties as "optional" in the documentation, then we should change the documentation, because this pattern is already very common.

Reply:
You are right. Let's change this to `boolean`.